ArcGISRuntimeSDKQt_QMLSamples/Search/OfflineGeocode/README.metadata.json

```diff
@@ -6,8 +6,8 @@
             "path": "~/ArcGIS/Runtime/Data/tpkx/streetmap_SD.tpkx"
         },
         {
-            "itemId": "c88c170e18f74740a0e4c660a5ff51c4",
-            "path": "~/ArcGIS/Runtime/Data/Locators/SanDiegoStreetAddress/SanDiego_StreetAddress.zip"
```

Comment:
did you test this out? I seem to remember some nuance in the sample viewer download codepath where if you download a zip file, it will unzip it automatically, whereas if you specify the individual file within the zip file, it might not behave as expected. I can't remember all the details exactly...

Reply:
@ldanzinger the C++ and QML implementations of Offline geocode differed in this. C++ looked for the `.loc` file and QML looked for the `.zip` file. Both workflows work, so in order to standardize the two, I went with the C++ workflow. If you feel that getting the `.zip` would be more stable we can opt for that instead.